content/docs/aws/aws-e2e.md

```diff
@@ -208,12 +208,13 @@ kfctl apply -f kfctl_aws_cognito.v1.0.0.yaml -V
 
 That shouldn't take a long time. There shouldn't by any errors, and when ready you can validate that you can see the kubeflow namespace.
 
-At this point you will also have an ALB, it takes around 3 minutes to be ready. When ready, copy the DNS name of that load balancer and create 3 CNAME entries to it in Route53:
+At this point you will also have an ALB, it takes around 3 minutes to be ready. When ready, copy the DNS name of that load balancer and create 2 CNAME entries to it in Route53:
 
-* `platform.domain.com`
 * `*.platform.domain.com`
 * `*.default.platform.domain.com`
```

Comment:
Is `*.default.platform.domain.com` needed?

Reply:
Yes, it's required for the next step, which is the inference service.

Reply:
Make sense.